test/cases/api.sh

```diff
@@ -53,8 +54,15 @@ case $CLOUD_PROVIDER in
   "$CLOUD_PROVIDER_AZURE")
     echo "Testing Azure"
     ;;
+  "$CLOUD_PROVIDER_AWS_S3")
+    echo "Testing S3 bucket upload"
+    if [[ $ID != "rhel" ]]; then
+        echo "Skipped. S3 upload test is only tested on RHEL."
```

Comment:
A tiny nitpick: I would love to see here an explanation why this is tested only on RHEL (the only distro supporting rhel-edge-commit). Someone will surely stumble upon this.

Reply:
Good thinking thanks.